go.mod

```diff
@@ -47,6 +48,7 @@ require (
 	github.com/golang/protobuf v1.3.2
 	github.com/google/go-github v17.0.0+incompatible
 	github.com/google/go-metrics-stackdriver v0.0.0-20190816035513-b52628e82e2a
+	github.com/google/go-querystring v1.0.0 // indirect
```

Comment:
Can you please ensure you're using Go 1.13 and `go mod tidy`?

Reply:
@jefferai Per your suggestion, we have rebased this PR on top of the `go-kms-wrapping` branch, ensured usage of go 1.13, and ran a `go mod tidy`.

These changes are available in our fork here: https://github.com/umg/vault/commit/7f86b76a64f40d250d2919523dd6016a280c0a6b

This is now working as expected in our environment, please let us know if there is any additional changes required for this to be merged into upstream.

Reply:
barring other feedback, it looks like you have some conflicts that need to be resolved as well.

Reply:
> Could you take a look a this? It's somewhat urgent for us at Jenkins-x and I'm happy to make any changes needed.

It's definitely a good feature we would like too! thanks for this @dgozalo 
@dgozalo would you have time to look at the remaining conflict and https://github.com/hashicorp/vault/pull/7738#discussion_r362996595?

Reply:
I'll have some more time to work on this again and prioritize it so it can be merged soon.